### ISSUE TITLE
Incompatibility with IntelliJ IDEA CE 2020.2

### ISSUE THREAD

**Issue by GiovanniBordiga** [halirutan/Mathematica-IntelliJ-Plugin#170]
Hello everyone!
I haven't found other issues about this problem so I'm reporting it here.

**Bug description**
Looks like the plugin is incompatible with the latest version of IntelliJ IDEA CE 2020.2.
At launch the following error message appears

```
Plugin "Wolfram Language" is incompatible (until build 201.SNAPSHOT < IC-202.6397.94).
```

**To Reproduce**
Steps to reproduce the behaviour:
1. Update to IntelliJ IDEA CE 2020.2
2. Launch IntelliJ IDEA

**Expected behaviour**
Plugin should be compatible with IntelliJ IDEA 2020.2.

**Versions:**
 - OS: Ubuntu 18.04
 - IntelliJ IDEA 2020.2 (Community Edition) Build #IC-202.6397.94
 - Wolfram Plugin Version 2020.1.1

Hope this is helpful. Thanks!

**Comment by halirutan** [comment]
Yes, this is correct. I've been working on some fixes and didn't realize that version 2020.2 is already out of beta. I'll be pushing an update in the next days.

**Comment by GiovanniBordiga** [comment]
> Yes, this is correct. I've been working on some fixes and didn't realize that version 2020.2 is already out of beta. I'll be pushing an update in the next days.

Thanks for the prompt reply. Looking forward for your update. Keep up the great work! :+1:

**Comment by szhorvat** [comment]
Please post a note in this thread when the update is pushed. I'm holding off on the IDEA update until I know it'll be compatible with the WL plugin :-)

**Comment by halirutan** [comment]
@GiovanniBordiga @szhorvat I pushed an update to the beta channel tonight. Adding the beta channel for plugins to your IDE is [only some clicks](https://wlplugin.halirutan.de/docs/installation/#using-eap-beta-releases) and it ensures that at least some other people except of me tried it before we push it to the main plugin channel.

It would be awesome if you tried it. Although I changed quite a bit, it should work as expected (but that's what I always say, right?). I think a quick check for one or two days should be enough and then we make it available to everyone.
